### PR TITLE
[Test framework MVP] PartialExportTest

### PR DESCRIPTION
**Pull request by lhanusov**
Part of #34494

### CODE REVIEW

File: tests/base/src/test/java/org/keycloak/test/utils/Assert.java
Comment (stianst) [inline tests/base/src/test/java/org/keycloak/test/utils/Assert.java:1]: See https://github.com/keycloak/keycloak/pull/34960/files#r1846026768

File: tests/base/src/test/java/org/keycloak/test/admin/partialexport/PartialExportTest.java
Comment (stianst) [inline tests/base/src/test/java/org/keycloak/test/admin/partialexport/PartialExportTest.java:1]: I'm not convinced we should have the `partialexport-testrealm.json`, that makes the test very hard to maintain. Would be ideal to do a RealmConfig instead. That would also stress the RealmConfigBuilder a bit as we'd need to expand on that to support more stuff.

File: tests/base/src/test/java/org/keycloak/test/utils/Assert.java
Comment (stianst) [inline tests/base/src/test/java/org/keycloak/test/utils/Assert.java:1]: Already added by @miquelsi here https://github.com/keycloak/keycloak/blob/main/tests/utils/src/main/java/org/keycloak/test/utils/Assert.java

File: tests/base/src/test/java/org/keycloak/test/admin/partialexport/PartialExportTest.java
Comment (stianst) [reply tests/base/src/test/java/org/keycloak/test/admin/partialexport/PartialExportTest.java:1]: Not sure what we want to do here, maybe we just leave it as is now, then we can revisit these json files later?

Comment (lhanusov) [reply tests/base/src/test/java/org/keycloak/test/admin/partialexport/PartialExportTest.java:1]: Good info, it is only used for this test class, bad info, it has more then 1000 lines, so re-writing it to the framework would take a lot of time with a huge java class. :-)

Comment (stianst) [reply tests/base/src/test/java/org/keycloak/test/admin/partialexport/PartialExportTest.java:1]: Yeah, let's just leave it

**Review by stianst — APPROVED** [review]

**Pull request merged**